packages/react-examples/src/react/__snapshots__/GroupedList.CustomCheckbox.Example.tsx.shot

```diff
@@ -910,7 +910,7 @@ exports[`Component Examples renders GroupedList.CustomCheckbox.Example.tsx corre
                                   }
                                 }
                               >
-                                item-0 Lorem ipsum dolor sit
+                                item-0 Lorem ipsum dolor sit amet
```

Comment:
I'm confused 😅😅 

How did TS upgrade change snapshots ?

Reply:
TS upgrade caused jest mocking to fail. This is a result of the implemented workaround with example data.